MANUALES.md

```
@@ -0,0 +1,298 @@
+## General Options
+- Display help for specific module or overall usage.
+
+```bash
+armbian-config [--help | -h | help] | [module_name]
+```
+List of all avalible features
+```bash
+Usage: ./bin/armbian-config [] [options]
+
+1. Toggle SSH lastlog
+	toggle_ssh_lastlog toggle_ssh_lastlog
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)